requirements/tools/Makefile

```diff
@@ -55,19 +63,23 @@ touch: ## touches all package requirement inputs
 	# Touchs all target input requirements
 	@$(foreach p,${_target-inputs},touch $(p);)
 
-reqs: _check_py36_version ## updates requirements of all package libraries
+reqs: ## updates requirements of all package libraries
```

Comment:
so you removed this check. why not having it for 3.8.10?  forgotten?

Reply:
this makefile runs inside the dockerfile which already installs the right version.

Reply:
ok, so you don't run it directly from the host anymore? it helped me some times when forgetting to change the python version. minor.